tomography_preprocessing/tilt_series_alignment/aretomo/batch.py

```diff
@@ -47,6 +48,8 @@ def batch_aretomo(
         console.log(f'ERROR: {e}')
         raise RuntimeError(e)
     console.log('Extracting metadata for all tilt series.')
+    tilt_series_directory, external_directory, alignments_directory = \
+        create_alignment_job_directory_structure(output_directory)
```

Comment:
I think here, all you need to do is create the external directory, and define the variable external_directory. This creates unneccessary tilt_series and alignment directories in the main folder otherwise